### PR TITLE
[Hub Generated] Review request for Microsoft.AzureStack to add version preview/2020-06-01-preview

### PR DESCRIPTION
**Pull request by ionuttulai**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/ionuttulai/azure-rest-api-specs/dev-azurestack-Microsoft.AzureStack-2020-06-01-preview-clean...master/).

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [x] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [x] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://msazure.visualstudio.com/One/_workitems/create/Scenario?templateId=d1c215ec-5d6a-4cb3-a193-95ddd0a1ba96&ownerId=85bb92fb-21fe-41ca-9309-47f2c5f57515). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ionuttulai** [comment]
This PR is addressing the s360 swagger completeness issues by adding the 2020-06-01-preview version.

**Comment by xiaoxuqi-ms** [comment]
@ionuttulai , could you please the link to S360 items?

**Comment by ionuttulai** [comment]
S360 for swagger completeness: 

https://vnext.s360.msftcloudes.com/blades/unifiedplatform?blade=AssignedTo:All~KPI:6dc4e485-2d1b-449c-a224-30b183391437~SLA:2~Forums:All~_loc:UnifiedPlatform&def=&tile=S360_ServiceId:9537bbc3-8945-4a80-a398-9e894170cc28~_loc:__key__UnifiedPlatform__6dc4e485-2d1b-449c-a224-30b183391437&global=4:9537bbc3-8945-4a80-a398-9e894170cc28

**Comment by xiaoxuqi-ms** [comment]
@ionuttulai , to address S360 completeness issues, I think the following operations should also be added in new version: 
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDPROVIDERS/FABRICLOCATIONS
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDRESOURCEGROUPS/LINKEDPROVIDERS/UPDATELOCATIONS/UPDATES
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDPROVIDERS/UPDATELOCATIONS
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDPROVIDERS/DEPLOYMENTS
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDPROVIDERS/REGIONHEALTHS
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDRESOURCEGROUPS/LINKEDPROVIDERS/NETWORKINTERFACES/
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDRESOURCEGROUPS/LINKEDPROVIDERS/VIRTUALMACHINES/INSTANCEVIEW
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDRESOURCEGROUPS/LINKEDPROVIDERS/DEPLOYMENTS/
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDPROVIDERS/FABRICLOCATIONS/SCALEUNITNODES
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDPROVIDERS/IMAGES
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDRESOURCEGROUPS/LINKEDPROVIDERS/UPDATELOCATIONS/UPDATES/UPDATERUNS
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDRESOURCEGROUPS/LINKEDPROVIDERS/DEPLOYMENTS/OPERATIONS
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDPROVIDERS/VIRTUALMACHINES
GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.AZURESTACK/LINKEDSUBSCRIPTIONS/LINKEDPROVIDERS/LOCATIONS/VMSIZES
For details, please refer to the ado item: https://msazure.visualstudio.com/Swagger%20KPIs/_workitems/edit/8188004

**Comment by ionuttulai** [comment]
There's an offline thread about adding exceptions for s360 items on resources nested after /linkedSubscriptions/{name}. This PR covers the tracked linkedSubscriptions resource type.

File: specification/azurestack/resource-manager/Microsoft.AzureStack/preview/2020-06-01-preview/LinkedSubscription.json [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LinkedSubscription"
+            }
+          },
+          "201": {
```
Comment (jorgecotillo) [inline specification/azurestack/resource-manager/Microsoft.AzureStack/preview/2020-06-01-preview/LinkedSubscription.json:215]: >"201" [](start = 10, length = 5)

Just to double check, this is meant to be a sync operation right?

Comment (ionuttulai) [reply specification/azurestack/resource-manager/Microsoft.AzureStack/preview/2020-06-01-preview/LinkedSubscription.json:215]: yes, it's a sync operation

**Comment by ArcturusZhang** [comment]
Breaking changes for go are introduced by previous PRs.

**Comment by xiaoxuqi-ms** [comment]
@msyyc could you help to have a look at this PR since it has CI-breakingchange-python

**Comment by msyyc** [comment]
For python ,the PR is ok

**Review by xiaoxuqi-ms — APPROVED** [review]

**Comment by ionuttulai** [comment]
@xiaoxuqi-ms I think this is ready to be merged

**Pull request merged**